features/adres.feature

```diff
@@ -111,82 +105,105 @@ Functionaliteit: Adresvelden vullen
+      | persoonIdentificatie | 4cbcf7fd-d788-4be5-8ebc-17decde26131                                      |
+      | fields               | verblijfplaats.adresregel2,verblijfplaats.adresregel3,verblijfplaats.land |
+      Dan bevat de persoon met burgerservicenummer '<burgerservicenummer>' de volgende verblijfplaats gegevens
+      | naam        | waarde        |
+      | adresregel2 | <adresregel2> |
 
       Voorbeelden:
       | omschrijving                           | burgerservicenummer | woonplaatsnaam | postcode | locatiebeschrijving        | adresregel2            |
       | woonplaatsnaam anders dan gemeentenaam | 999992922           | Scheveningen   | 2583XL   |                            | 2583 XL  SCHEVENINGEN  |
       | geen woonplaatsnaam                    | 999990639           |                | 2584BZ   |                            | 2584 BZ  'S-GRAVENHAGE |
       | locatiebeschrijving                    | 000009921           |                |          | Woonboot in de Grote Sloot | 'S-GRAVENHAGE          |
 
-
   Rule: Voor een buitenlands adres worden de adresregels en land gevuld uit de gegevens over verblijf buitenland (groep 13)
 
     Scenario: persoon verblijft in het buitenland
-      Gegeven persoon met burgerservicenummer "999993483" heeft de volgende gegevens bij verblijfplaats:
-      | gemeente van inschrijving (09.10) | 1999            |
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde                               |
+      | identificatie       | 4cbcf7fd-d788-4be5-8ebc-17decde26131 |
+      | burgerservicenummer | 999993483                            |
+      En de persoon heeft de volgende verblijfplaats gegevens
+      | naam                              | waarde          |
+      | gemeente van inschrijving (09.10) | 1999            | # kan dit ingevuld zijn voor een buitenlands adres?
       | land adres buitenland (13.10)     | 5010            |
       | regel 1 adres buitenland (13.30)  | Rue du pomme 25 |
       | regel 2 adres buitenland (13.40)  | Bruxelles       |
       | regel 3 adres buitenland (13.50)  | postcode 1000   |
-      En element woonplaatsnaam (11.70) heeft geen waarde
-      Als persoon met burgerservicenummer "{burgerservicenummer}" wordt opgevraagd met fields=verblijfplaats.woonplaats,verblijfplaats.gemeenteVanInschrijving
-      Dan bevat in het antwoord verblijfplaats:
-      """
-      {
-        "adresregel1": "Rue du pomme 25",
-        "adresregel2": "Bruxelles",
-        "adresregel3": "postcode 1000",
-        "land": {
-            "code": "5010",
-            "omschrijving": "België"
-        }
-      }
-      """
+      | woonplaatsnaam (11.70)            |                 | # is dit een veld waarop moet worden gecheckt voor buitenlands adres? Is het gevuld zijn van één of meerdere groep 13 velden niet voldoende?
```

Comment:
> Is het gevuld zijn van één of meerdere groep 13 velden niet voldoende?

klopt. woonplaatsnaam kan nooit gevuld zijn in combinatie met groep 13

Reply:
Dus woonplaatsnaam kan worden verwijderd uit deze gegeven, omdat het niet relevant is bij het bepalen van verblijfplaats buitenland? En hoe zit het met gemeente van inschrijving? Kan die gevuld zijn bij een verblijfplaats buitenland?

Reply:
>  En hoe zit het met gemeente van inschrijving? Kan die gevuld zijn bij een verblijfplaats buitenland?

Ja die is altijd gevuld, ook bij een verblijf buitenland, en zal dan de waarde 1999 "Registratie Niet Ingezetenen (RNI)" hebben. Wanneer er geen gemeenteVanInschrijving is (zelfs niet RNI), is de persoon niet ingeschreven. Dus in onze API heeft per definitie elke persoon een gemeenteVanInschrijving.

Reply:
> Dus woonplaatsnaam kan worden verwijderd uit deze gegeven, omdat het niet relevant is bij het bepalen van verblijfplaats buitenland?

Ja

Reply:
ok. aangepast